### ISSUE TITLE
Add Telonics capability

### ISSUE THREAD

**Issue by annam21** [Huh/collar#64]
Feature request: add Telonics to the list of companies

**Comment by Huh** [comment]
I hear you and will keep this high on the list. I spoke with Telonics at a conference last year and it raised some concerns about how to interact with their data. My questions at this point are

- [ ] Do they have an API similar to the others?
- [ ] Is external software required?
- [ ] I think there is a key like file that is needed to interact with the data, need clarification on that

I have a vague recollection that they do not have an API and are generally not setup for a third party to interact with data, but maybe they have some proprietary software that we could potentially use system calls on to interact with "programmatically". I am not saying that is correct. I/we should follow up with them to see if we can get some clarification.

**Comment by foresthayes** [comment]
I have some (limited) insight on a couple of those points –

The primary method of data access is through stand-alone software (https://www.telonics.com/software/tdc.php). I believe an account with a username and password is the only option for interacting with the program (no key files).

There is a limited ability to access data from a web browser (https://www.telonics.com/software/status.php) – but data are limited to the last location for each collar. Depending on the use case, web-scraping these data could provide a stopgap solution. 

I am not sure if this is feasible, but it may be worth looking into retrieving data directly from MetOcean. MetOcean provides the backend service for Telonics (Iridium collars) so it may be possible to access data from them using an API. That said, I don’t know if MetOcean stores any historic data or if that is only available through Telonics.

**Comment by tpetroel** [comment]
Any more recent updates on this? We still occasionally have Telonics collars in the mix and would be great to see them integrated for access here as opposed to going through MoveBank or TDC.

**Comment by Huh** [comment]
Agreed. I think the trouble is we need to rely on external software to make it happen. If you would like to email me I can provide a workflow from another collaborator that may accomplish what you want.
